docs/directions.js

```diff
@@ -1,27 +0,0 @@
-$(function () {
-
```

Comment:
My opinion is that you want someone who is at the top of the file who is reading it for the first time to understand within a couple seconds what is happening in the file. Here, the first thing the file does is to register the jQuery selectors. I would make the top of the file read:

```
this.registerSelectors();
this.doStep2();
this.doStep3();
...
```